packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +39,11 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        if (externalPkg && externalPkg.name) {
+            const { name, version } = require('../../../' + externalPkg.name + '/package.json');
```

Comment:
Use `require(`${externalPkg.name}`/package.json)`, also we need try catch for that, because not all package installed by defualt